### ISSUE TITLE
[Fix] [forecasting] low accuracy when backtesting because of correlation of ODE parameters are not considered

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#786]
## Summary
In forecasting, accuracy is quite low (i.e. predicted number of cases is far from actual values) when backtesting with Italy data and today=24Apr2021 because of correlation of ODE parameters are not considered. As failed in #778, ODE parameter values have correlation. This is related to #778 and #783.

## Codes

```Python
import covsirphy as cs
# Dataset preparation
data_loader = cs.DataLoader("input")
jhu_data = data_loader.jhu()
oxcgr_data = data_loader.oxcgrt()
# Scenario analysis
snl = cs.Scenario(country="Italy")
snl.register(jhu_data, extras=[oxcgrt_data]))
snl.timepoints(today="24Apr2021")
snl.trend()
snl.estimate(cs.SIRF)
snl.fit()
snl.predict()
snl.history("Confirmed")
snl.history("Infected")
snl.history("Recovered")
snl.history("Fatal")
```

## Outputs

![ita_10_history_Confirmed](https://user-images.githubusercontent.com/7270139/119234744-5f9d3d80-bb6a-11eb-8664-9f797a0733d3.png)
![ita_11_history_Infected](https://user-images.githubusercontent.com/7270139/119234746-6035d400-bb6a-11eb-95d4-833d8b640ff9.png)
![ita_12_history_Recovered](https://user-images.githubusercontent.com/7270139/119234747-60ce6a80-bb6a-11eb-8571-a2b729c27f58.png)
![ita_13_history_Fatal](https://user-images.githubusercontent.com/7270139/119234748-61670100-bb6a-11eb-8937-0fa37c8482a1.png)

## Environment

- CovsirPhy version: 2.20.1-alpha
- Python version: 3.9.5
- Installation: poetry
- System: WSL (Ubuntu)

**Comment by lisphilar** [comment]
In sklearn, Decision Tree Regressor accepts multioutput (y=ODE parameter values), considering correlation of y values. We can remove `MultioutputRegressor` in decision tree regression.
https://scikit-learn.org/stable/modules/tree.html#multi-output-problems

**Comment by lisphilar** [comment]
With #787, accuracy was improved especially for Confirmed/Recovered, which leads major improvement in Infected.
![ita_10_history_Confirmed](https://user-images.githubusercontent.com/7270139/119235135-f4546b00-bb6b-11eb-95f2-93b39d36c89f.png)
![ita_11_history_Infected](https://user-images.githubusercontent.com/7270139/119235136-f4ed0180-bb6b-11eb-943d-83798505d451.png)
![ita_12_history_Recovered](https://user-images.githubusercontent.com/7270139/119235137-f5859800-bb6b-11eb-8248-2970aacbdead.png)
![ita_13_history_Fatal](https://user-images.githubusercontent.com/7270139/119235138-f5859800-bb6b-11eb-82c8-e3c6bf591b69.png)

**Comment by lisphilar** [comment]
Because not completed in Confirmed/Fatal (-> Infected), we need to continue efforts in new issues.
